### PR TITLE
Stop spinning on error.

### LINKED ISSUE #54: Stop the spinner on error
Currently, when a server isn't responding and an error is displayed after a form submission, the spinner is still spinning. We should stop it when an error occurs.

### PR DESCRIPTION
**Pull request by almet**
Fix #54

### CODE REVIEW

File: formbuilder/actions/server.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import config from "../config";
 export const FORM_PUBLISH = "FORM_PUBLISH";
 export const FORM_PUBLICATION_PENDING = "FORM_PUBLICATION_PENDING";
 export const FORM_PUBLICATION_DONE = "FORM_PUBLICATION_DONE";
+export const FORM_PUBLICATION_FAILED = "FORM_PUBLICATION_DONE";
```
Comment (Natim) [inline formbuilder/actions/server.js]: FORM_PUBLICATION_FAILED = "FORM_PUBLICATION_FAILED";

File: formbuilder/components/builder/Form.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export default function EditableForm(props) {
   let saveButtonValue = "Create form";
   if (props.status == "pending") {
     saveButtonValue = <div>Create form <i className="spin glyphicon glyphicon-refresh" /></div>;
-  } else if (props.status == "done"){
+  } else if (props.status == "done") {
```
Comment (Natim) [inline formbuilder/components/builder/Form.js]: We could add a rule for props.status "failed" with ``glyphicon-remove``

Comment (almet) [inline formbuilder/components/builder/Form.js]: I actually tried but I'm not sure it's valuable :the error is already shown to the user and the button is here to do an action, having a cross in it is counter intuitive

File: formbuilder/actions/server.js (hunk 1 above)
Comment (almet) [inline formbuilder/actions/server.js]: Thanks!

**Pull request merged**